src/vma/util/sys_vars.cpp

```diff
@@ -110,6 +110,7 @@ namespace vma_spec {
 	static const char *spec_names_mcd_irq[]   = {"mcd-irq", "624", NULL};
 	static const char *spec_names_rti[]       = {"rti", "784", NULL};
 	static const char *spec_names_6973[]      = {"6973", NULL};
+	static const char *spec_names_stac[]      = {"stac", NULL};
```

Comment:
I don't think we should add numeration and textual support for new specification, it might be ambiguous and confusing. Also it wasn't in use till now, so we don't need to supply "backward compatibility". 

but it's your call